src/main/groovy/edu/wpi/first/gradlerio/frc/FRCNativeArtifact.groovy

```diff
@@ -27,7 +27,9 @@ class FRCNativeArtifact extends NativeArtifact {
         targetPlatform = NativePlatforms.roborio
 
         predeploy << { DeployContext ctx ->
+            def binFile = PathUtils.combine(ctx.workingDir, filename ?: file.get().name)
             ctx.execute(". /etc/profile.d/natinst-path.sh; /usr/local/frc/bin/frcKillRobot.sh -t 2> /dev/null")
+            ctx.execute("rm -f \"${binFile}\"")
```

Comment:
This already happens in the working dir, we shouldn't have to use PathUtils.combine

Reply:
Isn't it running in admin? Wouldn't that make working dir /home/admin? I'm always a bigger fan of full paths anyway. Less to go wrong.

Reply:
We override the working dir in the RoboRIO target: https://github.com/wpilibsuite/GradleRIO/blob/master/src/main/groovy/edu/wpi/first/gradlerio/frc/RoboRIO.groovy#L22

Reply:
At the moment it's a no-op since you're depending on workingDir anyway, workingDir by definition is the dir you're working in :P